From 1203808f59614f3bace1631cc713dcaa89026dde Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Mon, 8 Aug 2022 14:19:36 -0400
Subject: [PATCH] Issue 5397 - Fix check pick error #2

Description:

Original commit included a free for repl conn creds which does not exist
in 1.4.3
---
 ldap/servers/plugins/replication/repl5_connection.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ldap/servers/plugins/replication/repl5_connection.c b/ldap/servers/plugins/replication/repl5_connection.c
index be8bba08e..b6bc21c46 100644
--- a/ldap/servers/plugins/replication/repl5_connection.c
+++ b/ldap/servers/plugins/replication/repl5_connection.c
@@ -247,7 +247,6 @@ conn_delete_internal(Repl_Connection *conn)
     slapi_ch_free_string(&conn->last_ldap_errmsg);
     slapi_ch_free((void **)&conn->hostname);
     slapi_ch_free((void **)&conn->binddn);
-    slapi_ch_free((void **)&conn->creds);
     slapi_ch_free((void **)&conn->plain);
 }
 
-- 
2.37.1

